### ISSUE TITLE
The reason for evaluation metric difference between your Implemented baseline and MS-TCN

### ISSUE THREAD

**Issue by houzhijian** [cmhungsteve/SSTDA#6]
Hi, steve:

     Do you think what is the reason for the improvement between your implemented baseline and reported result in the original MS-TCN paper?

**Comment by cmhungsteve** [comment]
I think it's mainly about the hyper-parameters.
I spent more time finding the best hyper-parameters (e.g. learning rate).
Also, different machines and libraries also affect the numbers a little bit.

**Comment by houzhijian** [comment]
Got it. Thank you for your response.
